frontend/src/routes/Dashboard/index.js

```diff
@@ -59,63 +58,83 @@ function Dashboard(props) {
       .then((result) => setData(result));
   }, []);
 
+  const userStatus = () => (
+    <div className={classNames(styles.statusList)}>
+      <div className={classNames("row", styles.statusItem)}>
+        <span
+          className={styles.dot}
+          style={{ background: statusMapping[story.sick].color }}
+        />
+        {statusMapping[story.sick].name.toUpperCase()}
+      </div>
+      <div className={classNames("row", styles.statusItem)}>
+        <span
+          className={styles.dot}
+          style={{ background: statusMapping[story.tested].color }}
+        />
+        {statusMapping[story.tested].name.toUpperCase()}
+      </div>
+      <div></div>
+    </div>
+  );
+
+  const latestUpdate = () => (
+    <div>
+      <h3>LATEST TOTALS</h3>
+      <div className="row">
+        <div className={classNames(styles.totalItem)}>
+          ACTIVES
+          <div className={classNames(styles.totalItemNum)}>
+            {data.confirmed && data.confirmed.toLocaleString()}
+          </div>
+        </div>
+        <div className={classNames(styles.totalItem)}>
+          DEATHS
+          <div className={classNames(styles.totalItemNum)}>
+            {data.deaths && data.deaths.toLocaleString()}
+          </div>
+        </div>
+        <div className={classNames(styles.totalItem)}>
+          RECOVERED
+          <div className={classNames(styles.totalItemNum)}>
+            {data.recovered && data.recovered.toLocaleString()}
+          </div>
+        </div>
+      </div>
+    </div>
+  );
+
+  const suggestions = () => (
+    <div>
+      <h3>SUGGESTIONS</h3>
+      <div>Stay at home</div>
```

Comment:
This `div` is only used to add a linebreak. Since we're tweaking this whole bit of code, I'd change it for a `p`: it's more semantic / describes better what the element is, and that's the whole point of HTML!

But bear in mind, `p` usually has a default `margin` / `padding` (I can never remember which!), so you'll probably have to clear that up in CSS with something that looks like

```CSS
.root p { margin: 0; }
```

Reply:
Sounds good